packages/@okta/i18n/src/properties/login.properties

```diff
@@ -815,7 +815,15 @@ deviceTrust.universalLink.fallback.getOktaMobile.title = Get Okta Mobile
 # {0} and {1} are beginning and closing HTML elements.
 deviceTrust.universalLink.fallback.getOktaMobile.subtitle = Go to the {0}AppStore{1}, {0}search{1} for {0}Okta Mobile{1} and tap on {0}GET{1} Okta Mobile. Once installed, sign in to Okta Mobile and follow the instructions to secure your device.
 
+# Fastpass
+core.auth.factor.signedNonce.error.invalidEnrollment=Your device or account was invalidated. Re-enroll your account on Okta Verify and try again.
+core.auth.factor.signedNonce.error.invalidFactor=Your device or account was invalidated. Re-enroll your account on Okta Verify and try again.
+core.auth.factor.signedNonce.error.invalidDevice=Your device or account was invalidated. If this is unexpected, contact your administrator for help.
+core.auth.factor.signedNonce.error.invalidDeviceStatus=Your device or account was invalidated. If this is unexpected, contact your administrator for help.
```

Comment:
In what scenarios is an invalidated account **expected** for users attempting to authenticate? Seems like we could leverage the `core.auth.factor.signedNonce.error.deletedDevice` key for each of these error cases.

Reply:
I will check if this can be done, It will require some core changes

Reply:
line 819 and 820 has the same text, we only need one

Reply:
updated